OVS_Bonding/README.md

```diff
@@ -0,0 +1,55 @@
+# <u>Setting up network bonding on specific interfaces during the ignition process</u>
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+- All raw files and scripts are located in the RawFiles folder.
+- This guide describes the customisations needed for installation when using a PXE server.
+<br>
+
+## Prerequisites
+- PXE server with ability to set kernel arguments.
+- Web accessible location(HTTP only) to upload the <i> capturemacs.ign </i> file .
+
+## Usage  
+
+### <U>1. Adding parameters to PXE server </u>
+Key   |  Type | Value | Exmaple |Discription
+---   | --- | --- | --- | --- |
+ignition.config.url  | String | http://{location reachable to PXE and remote VM in order to get the ignition file}/capturemacs.ign custom-config | ignition.config.url=http://my.web.server.redhat.com/capturemacs.ign custom-config | This parameter is used to allow the server to load an extra ignition file which captures the MAC addresses and saves them into a file for later use in  machine config.
+macAddressList | String | List of MAC address should be set per host In the following order: <br> 1. Primary NIC <br> 2. Secondary NIC <br> 3. All the rest (optional)| 00:01:02:03:04:05,06:07:08:09:10:11 ... | This will be the list of MAC addresses which exist on the host and will be grabbed by the ignition file.
```

Comment:
`3. All the rest (optional)` - IIUC we don't do anything with #3+ macs... are we sure we want the user to insert them anyway?